packages/core/src/extensionShared.ts

```diff
@@ -59,13 +58,18 @@ let localize: nls.LocalizeFunc
  * Activation/setup code that is shared by the regular (nodejs) extension AND web mode extension.
  * Most setup code should live here, unless there is a reason not to.
  */
-export async function activateShared(context: vscode.ExtensionContext, contextPrefix: string): Promise<ExtContext> {
+export async function activateShared(
+    context: vscode.ExtensionContext,
+    contextPrefix: string,
```

Comment:
In the future, if we have to add more parameters we'll want to create a single object with all these params instead.

Reply:
like an args: {} of sorts? Agreed